pkg/util/cache/cache_test.go

```diff
@@ -125,6 +125,20 @@ func TestLRUCache(t *testing.T) {
 		})
 		assert.Equal(t, ErrNoSuchItem, err)
 	})
+
+	t.Run("test reloader", func(t *testing.T) {
```

Comment:
Do we need a dedicated test on `MarkItemNeedReload` under concurrent conditions?

Reply:
> Do we need a dedicated test on `MarkItemNeedReload` under concurrent conditions?

ok, I will design a case.